BUILDIT.md

```diff
@@ -113,14 +113,21 @@ With that in place, let's get building!
   - [14.3 Building the Stats Page](#143-building-the-stats-page)
   - [14.4 Broadcasting to `stats` channel](#144-broadcasting-to-stats-channel)
   - [14.5 Adding tests](#145-adding-tests)
-- [15. `People` in Different Timezones 🌐](#15-people-in-different-timezones-)
-  - [15.1 Getting the `person`'s Timezone](#151-getting-the-persons-timezone)
-  - [15.2 Changing how the timer datetime is displayed](#152-changing-how-the-timer-datetime-is-displayed)
-  - [15.3 Persisting the adjusted timezone](#153-persisting-the-adjusted-timezone)
-  - [15.4 Adding test](#154-adding-test)
-- [16. Run the _Finished_ MVP App!](#16-run-the-finished-mvp-app)
-  - [16.1 Run the Tests](#161-run-the-tests)
-  - [16.2 Run The App](#162-run-the-app)
+- [15. Adding Sorting to the track metrics table and more additional fields](#15-adding-sorting-to-the-track-metrics-table-and-more-additional-fields)
```

Comment:
@panoramix360 these additions are great.
Please copy and paste them into the new file: 
And create a _separate_ PR for the docs. 
As per https://github.com/dwyl/mvp/issues/397

Reply:
Added there:
https://github.com/dwyl/book/pull/96

Is that what you meant?